Sources/GRPC/GRPCProtobufPayload.swift

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import NIO
+import SwiftProtobuf
+
+public protocol GRPCProtobufPayload: GRPCPayload {}
+
+extension GRPCProtobufPayload where Self: SwiftProtobuf.Message {
+    init(serializedByteBuffer: inout NIO.ByteBuffer) throws {
+        try self.init(serializedData: serializedByteBuffer.readData(length: serializedByteBuffer.readableBytes)!,
+                  extensions: nil,
+                  partial: false,
+                  options: BinaryDecodingOptions())
+    }
+    func serialize(into buffer: inout NIO.ByteBuffer) throws {
+        let data = try self.serializedData()
+        buffer.writeBytes(data)
+    }
+}
```

Comment:
Never being called, since the code relies on the GRPCPayload instead of this protocol

Reply:
Added some comments to help fix this :)